cola/widgets/toolbar.py

```diff
@@ -0,0 +1,527 @@
+        ACTION_FIND_FILES: lambda: finder.finder(),
+        ACTION_EDIT_REMOTE_REPO: lambda: editremotes.remote_editor().exec_(),
+        ACTION_RECENT_EDIT: lambda: recent.browse_recent_files(),
+        ACTION_APPLY_PATCHS: lambda: patch.apply_patches(),
+        ACTION_EXPORT_PATCHS: lambda: guicmds.export_patches(),
+        ACTION_SAVE_TAR_ZIP: lambda: app().activeWindow().save_archive(),
+        ACTION_PREFERENCES: lambda: app().activeWindow().preferences()
+    },
+    N_('Actions'): {
+        ACTION_FETCH: lambda: remote.fetch(),
+        ACTION_PULL: lambda: remote.pull(),
+        ACTION_PUSH: lambda: remote.push(),
+        ACTION_STASH: lambda: stash.stash(),
+        ACTION_CREATE_TAG: lambda: createtag.create_tag(),
+        ACTION_CHERRY_PICK: lambda: guicmds.cherry_pick(),
+        ACTION_MERGE: lambda: merge.local_merge(),
+        ACTION_ABORT_MERGE: lambda: merge.abort_merge(),
+        ACTION_RESET_HEAD: lambda: guicmds.reset_branch_head(),
+        ACTION_RESET_WORKTREE: lambda: guicmds.reset_worktree(),
+        ACTION_GREP: lambda: grep.grep(),
+        ACTION_SEARCH: lambda: search.search()
+    },
+    N_('Commit@@verb'): {
+        ACTION_AMMEND_LAST: cmds.run(cmds.AmendMode, True),
+        ACTION_STAGE: lambda: action.ActionButtons.stage(
+            action.ActionButtons()),
+        ACTION_STAGE_ALL: cmds.run(cmds.StageUntracked),
+        ACTION_UNSTAGE_ALL: cmds.run(cmds.UnstageAll),
+        ACTION_UNSTAGE: lambda: action.ActionButtons.unstage(
+            action.ActionButtons()),
+        ACTION_LOAD_COMMIT: lambda: guicmds.load_commitmsg(),
+        ACTION_APPLY_COMMIT: cmds.run(cmds.LoadCommitMessageFromTemplate)
+    },
+    N_('Diff'): {
+        ACTION_EXPRESSION: lambda: guicmds.diff_expression(),
+        ACTION_BRANCHES: lambda: compare.compare_branches(),
+        ACTION_DIFFSTAT: cmds.run(cmds.Diffstat)
+    },
+    N_('Branch'): {
+        ACTION_REVIEW: lambda: guicmds.review_branch(),
+        ACTION_CREATE: lambda: createbranch.create_new_branch(),
+        ACTION_CHECKOUT: lambda: guicmds.checkout_branch(),
+        ACTION_DELETE: lambda: guicmds.delete_branch(),
+        ACTION_DELETE_REMOTE: lambda: guicmds.delete_remote_branch(),
+        ACTION_RENAME: lambda: guicmds.rename_branch(),
+        ACTION_BROWSE_CURRENT: lambda: guicmds.browse_current(),
+        ACTION_BROWSE_OTHER: lambda: guicmds.browse_other(),
+        ACTION_VISUALIZE_CURRENT: cmds.run(cmds.VisualizeCurrent),
+        ACTION_VISUALIZE_ALL: cmds.run(cmds.VisualizeAll)
+    },
+    N_('Rebase'): {
+        ACTION_REBASE: lambda: app().activeWindow().rebase_start(),
+        ACTION_REBASE_EDIT: lambda: app().activeWindow().rebase_edit_todo(),
+        ACTION_REBASE_CONT: lambda: app().activeWindow().rebase_continue(),
+        ACTION_REBASE_IGNORE: lambda: app().activeWindow().rebase_skip(),
+        ACTION_REBASE_ABORT: lambda: app().activeWindow().rebase_abort()
+    }
+}
+TOOLBAR_TEXTS = {
```

Comment:
I will try to remove inline imports. Yes integers are written in configuration instead translated strings.